src/main/resources/db/migration/V6.1.0.1__9686-move-harvestingclient-id.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE dvobject ADD COLUMN IF NOT EXISTS harvestingclient_id BIGINT;
+
+update dvobject dvo set harvestingclient_id = s.harvestingclient_id from
+(select id, harvestingclient_id from dataset d where d.harvestingclient_id is not null) s
+where s.id = dvo.id; 
+
```

Comment:
We also need to populate this harvestingclient_id for any existing harvested file dvobjects here. ... and it should be a relatively simple query, now that all the dataset-type dvobjects have the harvestingclient_id populated.

Reply:
@landreev the script update is in. I also added a release note which may not be necessary or could use your elaboration if you'd like. Thanks!

Reply:
I made a PR into this PR so we can add the FK @sekmiller, I spoke earlier with @landreev and it makes sense to add it since it was previously.